lib/utils/testem-events.js

```diff
@@ -121,6 +121,15 @@ class TestemEvents {
     }
   }
 
+  /**
+   * Record the module run details to the stateManager
+   *
+   * @param {*} metaData
+   */
+  recordModuleMetaData(metaData) {
+    this.stateManager.addMetadataToModuleMetadata(metaData);
```

Comment:
After reading this PR, is there a reason for us to store this metaData on the stateManager, or simply on this test-events object. 

My thoughts:

1) it seems to only be used by this object, shuttling the values to the second object seems redundant
2) it doesn't seems related to the state management itself, largely just a way of serializing data er: event meta data.

Reply:
My thought on storing the metaData on the stateManager is due to
- I consider the module detail as a part of test state
- and module details do not really seem to belong to testem-event where testem-event object is to coordinate testem events.. 

but open to your thoughts